messaging/src/main/java/org/axonframework/common/BuilderUtils.java

```diff
@@ -74,6 +74,17 @@ public static void assertPositive(int integer, String exceptionMessage) {
         assertThat(integer, number -> number >= 0, exceptionMessage);
     }
 
+    /**
+     * Assert that the given {@code value} is positive, meaning greater than, or equal to, zero. If not, an
+     * {@link AxonConfigurationException} is thrown containing the provided {code exceptionMessage}.
+     *
+     * @param integer the value to assert
+     * @param exceptionMessage the message for the exception
+     */
+    public static void assertPositive(long integer, String exceptionMessage) {
```

Comment:
Great addition to the `BuilderUtils`, but could you add a test for this? :-)

Reply:
Done

Reply:
`long integer` seems a bit odd :)

Reply:
Good point!

Reply:
Fixed already...